framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/DbNamingConventionRewriterExtensions.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using EFCore.NamingConventions.Internal;
+using Microsoft.EntityFrameworkCore;
+
+namespace Volo.Abp.EntityFrameworkCore
+{
+    public static class DbNamingConventionRewriterExtensions
+    {
+
+        public static void NamingConventionsRewriteName(this DbContextOptionsBuilder optionsBuilder,
+            DbNamingConvention namingConvention = DbNamingConvention.Default,
+            CultureInfo culture = null)
+        {
+            switch (namingConvention)
+            {
+                case DbNamingConvention.Default:
+                    break;
+                case DbNamingConvention.SnakeCase:
+                    optionsBuilder.UseSnakeCaseNamingConvention(culture);
+                    break;
+                case DbNamingConvention.LowerCase:
+                    optionsBuilder.UseLowerCaseNamingConvention(culture);
+                    break;
+                case DbNamingConvention.UpperCase:
+                    optionsBuilder.UseUpperCaseNamingConvention(culture);
+                    break;
+                case DbNamingConvention.UpperSnakeCase:
+                    optionsBuilder.UseUpperSnakeCaseNamingConvention(culture);
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        public static void NamingConventionsRewriteName(this DbContextOptionsBuilder optionsBuilder,
+            string namingConvention = nameof(DbNamingConvention.Default),
+            CultureInfo culture = null)
+        {
+            switch (namingConvention)
+            {
+                case nameof(DbNamingConvention.Default):
+                    break;
+                case nameof(DbNamingConvention.SnakeCase):
+                    optionsBuilder.UseSnakeCaseNamingConvention(culture);
+                    break;
+                case nameof(DbNamingConvention.LowerCase):
+                    optionsBuilder.UseLowerCaseNamingConvention(culture);
+                    break;
+                case nameof(DbNamingConvention.UpperCase):
+                    optionsBuilder.UseUpperCaseNamingConvention(culture);
+                    break;
+                case nameof(DbNamingConvention.UpperSnakeCase):
+                    optionsBuilder.UseUpperSnakeCaseNamingConvention(culture);
+                    break;
+                default:
+                    break;
+            }
+        }
```

Comment:
You can try to convert `(namingConvention )string`to `DbNamingConvention`, and then reuse `NamingConventionsRewriteName `method.